docs/source/how-to/deploy-dask-clusters.rst

```diff
@@ -6,8 +6,23 @@ locally on your own machine or on a distributed cluster.  If you are just
 getting started, then this page is unnecessary.  Dask does not require any setup
 if you only want to use it on a single computer.
 
+You can continue reading or watch the screencast below:
+
+.. raw:: html
+
+   <iframe width="560"
+           height="315"
+           src="https://www.youtube.com/embed/TQM9zIBzNBo"
+           style="margin: 0 auto 20px auto; display: block;"
+           frameborder="0"
+           allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+           allowfullscreen></iframe>
+
 Dask has two families of task schedulers:
 
+.. image:: ../images/dask-overview.png
```

Comment:
I'd move this image below the scheduler list (as you did in the `index`), so that the types of schedulers immediately follows the above intro "Dask has two families of task schedulers"

Reply:
Not sure what the stylistic guidelines are on this, but it might make sense to outline the schedulers section in the image.

Reply:
```suggestion
.. image:: ../images/dask-overview.png
   :alt: Dask is composed of three parts. "Collections" create "Task Graphs" which are then sent to the "Scheduler" for execution. There are two types of schedulers that are described in more detail below. 
```